### DISCUSSION TITLE
Flames Version 4.1 "Spicy-Ass Explosion"

### DISCUSSION THREAD

**Discussion by SeveralCircles** [colorfulFlames/flames#19] [Announcements]
## What's Changed
* Feature/flames api by @SeveralCircles in https://github.com/SeveralCircles/flames/pull/11
* This branch has outlived it's own welcome but oh well by @SeveralCircles in https://github.com/SeveralCircles/flames/pull/13
* Feature/today by @SeveralCircles in https://github.com/SeveralCircles/flames/pull/15
* Release/4.0 by @SeveralCircles in https://github.com/SeveralCircles/flames/pull/16
* V4.1 by @SeveralCircles in https://github.com/SeveralCircles/flames/pull/18

**Full Changelog**: https://github.com/SeveralCircles/flames/compare/v3.3.2...v4.1

<hr /><em>This discussion was created from the release <a href='https://github.com/SeveralCircles/flames/releases/tag/v4.1'>Flames Version 4.1 "Spicy-Ass Explosion"</a>.</em>

**Comment by Minotaurinmaze** [comment]
I love this version 
![image](https://user-images.githubusercontent.com/65237985/137509810-40a6aebe-0552-4a38-b73a-2ce3e9ada13a.png)
